public/normal_hlsc_sanger_2019/meta_study.txt

```diff
@@ -0,0 +1,9 @@
+type_of_cancer: BOWEL

```

Comment:
@ritikakundra how do you recommend to handle this? Should we add a "Normal" node to oncotree?